build/wix/Localization/mixxx_en-us.wxl

```diff
@@ -24,6 +24,7 @@
    <String Id="bitWidthMismatch32">You are about to install the 32-bit version but a 64-bit one is already installed. This can lead to undesired behavior. Please un-install the existing Mixxx first, or download the 64-bit installation package to upgrade (unless you know exactly what you are doing.)</String>
    <String Id="bitWidthMismatch64">You are about to install the 64-bit version but a 32-bit one is already installed. This can lead to undesired behavior. Please un-install the existing Mixxx first, or download the 32-bit installation package to upgrade (unless you know exactly what you are doing.)</String>
    <String Id="OlderVersionFound">An older version of Mixxx has been detected. As we changed installer, we can not smoothly upgrade. Please uninstall older version before installing this one.</String>
+   <String Id="VCredistNotFound">Installer was not able to find Microsoft Visual C++ 2015 Redistributable on this system. This is likely an installer bug that should be reported to Mixxx developpers.</String>
```

Comment:
> I also updated translation source on transifex.

Translating got a bit harder with the ampersands and variables.
In the past, we had conflicting shortcuts with some languages, because translators didn't fully understand what the ampersands do.

Unfortunately, the Windows installer can not share our extensive translation memory and glossary with the rest of the project (application, manual, website) because the base language is ``en_us`` instead of ``en``.

The installer got it right IMO, we need to carry over the base language for the other parts of the project to ``en_us``. Will check the requirements and submit a PR.